mappings/net/minecraft/entity/ai/memory/MemoryManager.mapping

```diff
@@ -0,0 +1,14 @@
+CLASS aiu net/minecraft/entity/ai/memory/MemoryManager
+	FIELD a hashMap Ljava/util/Map;
+	FIELD b linkedHashMap Ljava/util/Map;
+	FIELD c treeMap Ljava/util/Map;
```

Comment:
This class is called `Brain`. It does much more than just manage memories. Also, these names should be more descriptive. You can remove this class since I mapped it in my PR (#533)